src/Evolution/Systems/GeneralizedHarmonic/GaugeSourceFunctions/DampedHarmonic.cpp

```diff
@@ -54,12 +54,13 @@ double time_deriv_of_roll_on_function(const double time, const double t_start,
 }
 }  // namespace DampedHarmonicGauge_detail
 
+namespace {
 template <size_t SpatialDim, typename Frame>
-void damped_harmonic(
+void damped_harmonic_impl(
```

Comment:
What do you gain by factoring into an impl?

Reply:
Nothing yet, but I'll be submitting a PR that adds a template parameter to either do the rollon gauge or not. I find having the template parameter in the public function makes the public functions harder to use, and so factoring into an impl will allow `damped_harmonic` and `damped_harmonic_rollon` functions to share the internals of the computation, using `if constexpr` to control flow a bit, while making the public interface (in my opinion) significantly easier to use. I find something like `damped_harmonic<true>(...)` much harder to understand than `damped_harmonic_rollon` :)